homeassistant/components/min_max/sensor.py

```diff
@@ -232,6 +236,11 @@ def icon(self):
         """Return the icon to use in the frontend, if any."""
         return ICON
 
+    @property
+    def state_class(self) -> SensorStateClass:
+        """Return the state class."""
+        return SensorStateClass.MEASUREMENT
```

Comment:
This is a property that returns a static value, and thus should have been set as a class attribute instead.

See: <https://developers.home-assistant.io/docs/core/entity#entity-class-or-instance-attributes>

Reply:
Ops I forgot about this.